applications/EmpireApplication/test_examples/CoSim_DevExamples/potential_flow_rigid_2D_wing_fsi/MainCoSim.py

```diff
@@ -0,0 +1,15 @@
+from __future__ import print_function, absolute_import, division  # makes KratosMultiphysics backward compatible with python 2.6 and 2.7
```

Comment:
The test example potential_flow_rigid_2D_wing_fsi is way too large. We are trying to keep the size of the repository small so save this test locally by you and only commit the small rhombus test.